src/collective/embeddedpage/browser/embeddedpage.py

```diff
@@ -17,6 +17,10 @@ def __call__(self):
         # we receive a utf-8 encoded string from requests
         # lxml expect unicode though
         content = safe_unicode(response.content)
-        subtree = lxml.html.fromstring(content).find('body')
-        self.embeddedpage = etree.tostring(subtree)
+        content = lxml.html.fromstring(content)
+        subtree = content.find('body')
+        if subtree is not None:
```

Comment:
I'd rather just use "if subtree" here. "is not None" is not necessary here.

Reply:
Or maybe just "if content.find('body')". No need to use a variable if it is only used once.